tests/test_dialogues.py

```diff
@@ -1,17 +1,23 @@
 import glob
 import json
 
-import io
 import jsonpickle
 import pytest
 
 from rasa_core import utils
 from rasa_core.domain import Domain
 from rasa_core.tracker_store import InMemoryTrackerStore
 from tests.utilities import tracker_from_dialogue_file
+from tests.conftest import DEFAULT_DOMAIN_PATH
 
+test_dialogues = sorted(glob.glob('data/test_dialogues/*json'))
```

Comment:
please just list the paths directly as for the domains. since you zip them later, this will break if there are more than for anyways